lib/bolt/config.rb

```diff
@@ -519,6 +519,10 @@ def modulepath=(value)
       @data['modulepath'] = value
     end
 
+    def plugin_cache
+      @data['plugin-cache']
```

Comment:
This should have a default of `{}` in the initializer, similar to `apply_settings`, `plugin_hooks`, etc.